core/src/main/java/org/apache/gravitino/storage/relational/mapper/postgresql/FilesetMetaPostgreSQLProvider.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.gravitino.storage.relational.mapper.postgresql;
+
+import static org.apache.gravitino.storage.relational.mapper.FilesetMetaMapper.META_TABLE_NAME;
+
+import org.apache.gravitino.storage.relational.mapper.FilesetMetaBaseSQLProvider;
+import org.apache.gravitino.storage.relational.po.FilesetPO;
+
+public class FilesetMetaPostgreSQLProvider extends FilesetMetaBaseSQLProvider {
+  @Override
+  public String softDeleteFilesetMetasByMetalakeId(Long metalakeId) {
+    return "UPDATE "
+        + META_TABLE_NAME
+        + " SET deleted_at = floor(extract(epoch from((current_timestamp - timestamp '1970-01-01 00:00:00')*1000)))"
```

Comment:
I guess this line is longer than 100 chars, can you please fix all of such problems?

Reply:
done.